index.html

```diff
@@ -1096,8 +1096,14 @@ <h2><dfn data-lt="implicitly release the pointer capture|implicitly releasing po
                 if the pointer supports hover, user agent MUST also send corresponding boundary events necessary
                 to reflect the current position of the pointer with no capture.</p>
             <p>When the <a>pointer capture target override</a> is no longer [=connected=] [[DOM]],
-                the <a>pending pointer capture target override</a> and <a>pointer capture target override</a> nodes SHOULD be cleared
-                and also a PointerEvent named {{GlobalEventHandlers/lostpointercapture}} corresponding to the captured pointer SHOULD be fired at the document.</p>
+                the <a>pending pointer capture target override</a> node SHOULD be cleared and
```

Comment:
Clearing the `pending target override` here would drop a `setPointerCapture()` call that is made just before the deletion of current `target override`.  Not doing anything to the pending state here should work fine, right?

Reply:
Good point, we should only clear the pending pointer capture target if it was the removed node - which we would do anyways from the following point. I can remove this here and move the note below both paragraphs as it only follows from the combination of updating both things.

Reply:
Done.